Policies/Kubernetes/container-no-privilege-escalation/azurepolicy.json

```diff
@@ -0,0 +1,50 @@
+{
+  "properties": {
+    "displayName": "[Preview] [PSP]: Do not allow container privilege escalation in Kubernetes cluster",
```

Comment:
Remove prefix [preview] and [psp]. When you prepare these policies as built-in definitions, [preview] need not be explicitly specified in the display name.

Reply:
Instead add metadata like below...
"metadata": {
      "version": "1.0.0-preview",
      "category": "Kubernetes PSP",
      "preview": true
    },

We need to check on category name with Liz. For now, you may use "Kubernetes PSP" for testing.